lib/rack/tracker/google_analytics/google_analytics.rb

```diff
@@ -32,14 +32,12 @@ def tracker
   end
 
   def tracker_options
-    @tracker_options ||= begin
-      tracker_options = {}
+    @tracker_options ||= {}.tap do |tracker_options|
       options.slice(*ALLOWED_TRACKER_OPTIONS).each do |key, value|
         if option_value = value.respond_to?(:call) ? value.call(env) : value
```

Comment:
yeah, but wish it worked :)
I tried it. It works when the proc returns a value, but when the proc returns nil, option_value gets set to a Proc object and this goes through the if condition. I added a spec for that [case](https://github.com/railslove/rack-tracker/blob/master/spec/handler/google_analytics_spec.rb#L57).

```
»  value = lambda{1}
=> #<Proc:0x007fd81da52fb0@(pry):2 (lambda)>
»  value.try(:call) || value
=> 1
»  value = lambda{nil}
=> #<Proc:0x007fd81fec4038@(pry):4 (lambda)>
»  value.try(:call) || value
=> #<Proc:0x007fd81fec4038@(pry):4 (lambda)>
```